spec/webvalve/manager_spec.rb

```diff
@@ -261,9 +271,9 @@
           subject.setup
         end
 
-        expect(WebMock).to have_received(:stub_request).with(:any, %r{\Ahttp://something\.dev})
-        expect(WebMock).to have_received(:stub_request).with(:any, %r{\Ahttp://something\-else\.dev})
-        expect(WebMock).to have_received(:stub_request).with(:any, %r{\Ahttp://other\.dev})
+        expect(WebMock).to have_received(:stub_request).with(:any, %r{\Ahttp://something\.dev#{url_suffix_pattern}})
+        expect(WebMock).to have_received(:stub_request).with(:any, %r{\Ahttp://something\-else\.dev#{url_suffix_pattern}})
+        expect(WebMock).to have_received(:stub_request).with(:any, %r{\Ahttp://other\.dev#{url_suffix_pattern}})
```

Comment:
Maybe I should mock ServiceUrlConverter here but I sort of value the integration quality. Maybe I should keep it for one test and mock it elsewhere?